### PR TITLE
Update text and remove more placeholders.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Fill in all the text on all the pages from text and discussion on #16, #17 and #18 and remove missing placeholder images in favour of moving stuff around and more links to Open Source Guides.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Merging this so I can show it to people more easily but I'd still like post-merge feedback from everyone and anyone and I'll follow up in another PR.

**Comment by bkeepers** [comment]
I'm not crazy about using the guides so much, but we can update with simpler illustrations later.

**Comment by MikeMcQuaid** [comment]
@bkeepers Yeh, I think for the main page the e.g. legal one is kinda hilariously out of place by the "holes" in the page annoyed me too much. I think on the maintainer one they are probably worth highlighting; even as is they are effectively summarising our guides.

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <header class="jumbotron text-center bg-gray-light mb-6 border-bottom px-3">
   <div class="container-lg position-relative">
     <h1 class="alt-h1">Open Source Friday</h1>
-    <p class="alt-lead text-gray mb-md-5 col-md-8 mx-auto"><%= @subtitle || "Contribute to open source every Friday, and help new contributors, maintainers,
-and companies do the same." %></p>
+    <p class="alt-lead text-gray mb-md-5 col-md-8 mx-auto">Take some time one day every week to invest in the open source software you already use and the community of people that make it great</p>
```
Comment (nayafia) [inline app/views/home/index.html.erb]: Tagline is a bit long and meandering. We could split it up, e.g.

```Once a week, every week.```
```Invest in the open source software you use and love to make it great.```

Comment (nayafia) [inline app/views/home/index.html.erb]: This also repeats line 12 below.

**Comment by nayafia** [comment]
I have comments on things that weren't changed in this PR, so I'm just gonna start another PR with my edits. (Collaborating on content is weird on GitHub!)

Overall I think these are great changes :thumbsup: Are we ready to loop in Steph/Erin to give us :eyes:? I especially think we could use Erin's help with the company content, I know it needs polish but I don't know how.

**Comment by MikeMcQuaid** [comment]
> I have comments on things that weren't changed in this PR, so I'm just gonna start another PR with my edits. (Collaborating on content is weird on GitHub!)

👍 Reviewed your new PR.

> Overall I think these are great changes 👍 Are we ready to loop in Steph/Erin to give us 👀? I especially think we could use Erin's help with the company content, I know it needs polish but I don't know how.

For non-GitHub people following along: these are people who work for GitHub who are our social media and business marketing experts.

I think after https://github.com/ossfriday/ossfriday/pull/30 is merged we should bring them in, yeh.

**Pull request merged**